src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.pkgdef

```diff
@@ -1,4 +1,3 @@
-#include "versions.h"
```

Comment:
I don't know if it's worth it - this will require the C preprocessor, which will require another dependency in the VS tools, and we'd have to change the image.

I might look to see if I can just move these to attributes - they are all ProvideObjectAttribute, so should be easy to mimic.